### ISSUE TITLE
The bus drives very weirdly

### ISSUE THREAD

**Issue by Zzeyy** [FedericoDeBona/publictransport#5]

**Comment by Zzeyy** [comment]
The way it parks LOL
![image](https://user-images.githubusercontent.com/97387354/214091032-e6b195b7-49d2-412c-902b-89dc40a277ae.png)

**Comment by FedericoDeBona** [comment]
Unfortunately there is no easy fix since all the driving is managed internally by the game. A solution you could try is to move the bus stop position and try again until you are satisfied.

**Comment by Zzeyy** [comment]
> Unfortunately there is no easy fix since all the driving is managed internally by the game. A solution you could try is to move the bus stop position and try again until you are satisfied.

Well, the main problem here is that sometimes buses disappear, do not spawn or the bus loses the driver by disappearing.

**Comment by Zzeyy** [comment]
Removing this problem would make the script actually very usable!

**Comment by Zzeyy** [comment]
I hope one day you'll take a look at this. 🙏

**Comment by VIRUXE** [comment]
Indeed, they have an odd driving style. They run lights and go through sidewalks.
